components/org.wso2.carbon.identity.api.user.approval/org.wso2.carbon.identity.rest.api.user.approval.v1/src/gen/java/org/wso2/carbon/identity/rest/api/user/approval/v1/MeApi.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.rest.api.user.approval.v1;
+
+import org.wso2.carbon.identity.rest.api.user.approval.v1.dto.*;
+import org.wso2.carbon.identity.rest.api.user.approval.v1.MeApiService;
+import org.wso2.carbon.identity.rest.api.user.approval.v1.factories.MeApiServiceFactory;
+
+import io.swagger.annotations.ApiParam;
+
+import org.wso2.carbon.identity.rest.api.user.approval.v1.dto.TaskDataDTO;
+import org.wso2.carbon.identity.rest.api.user.approval.v1.dto.ErrorDTO;
+import org.wso2.carbon.identity.rest.api.user.approval.v1.dto.TaskSummeryDTO;
+import org.wso2.carbon.identity.rest.api.user.approval.v1.dto.StateDTO;
+
+import java.util.List;
+
+import java.io.InputStream;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.*;
+
+@Path("/me")
+
+
+@io.swagger.annotations.Api(value = "/me", description = "the me API")
+public class MeApi  {
+
+   private final MeApiService delegate = MeApiServiceFactory.getMeApi();
```

Comment:
@Autowired?

Reply:
Added in 9c7a2de6ec4ce3b19e5e0126858a3924c6a4b7c8